lesson10/index.js

```diff
@@ -0,0 +1,29 @@
+const Product = {
+  name: 'Кроссовки',
+  prise: 300,
```

Comment:
```suggestion
  price: 300,
```

опечатка в названии свойства